pom.xml

```diff
@@ -14,6 +14,7 @@
         <jenkins.version>2.121.3</jenkins.version>
         <java.level>8</java.level>
         <powermock.version>1.7.4</powermock.version>
+        <hpi.compatibleSinceVersion>3.0</hpi.compatibleSinceVersion>
```

Comment:
I believe this is still needed - it lets Jenkins know that there was a config format change in version 3.0 of the plugin

Reply:
Sorry to say, I don't understand your comment. So let's explain my change in more detail. Using the property `hpi.compatibleSinceVersion` is the short form of configuring the plugin via:
```
<plugin>
    <groupId>org.jenkins-ci.tools</groupId>
    <artifactId>maven-hpi-plugin</artifactId>
    <extensions>true</extensions>
    <configuration>
        <compatibleSinceVersion>3.0</compatibleSinceVersion>
    </configuration>
</plugin>
```
See its documentation https://jenkinsci.github.io/maven-hpi-plugin/hpi-mojo.html#compatibleSinceVersion and the CHANGELOG https://github.com/jenkinsci/maven-hpi-plugin/tree/maven-hpi-plugin-3.9#32-2019-01-16.
You can verify that the output is the same by looking inside the generated `META-INF/MANIFEST.MF`, there should still the entry `Compatible-Since-Version: 3.0`.

Reply:
d'oh! My brain translated this line as a deletion instead of addition.